package s2java;

public class exception {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
try {
	int data=100/0;
	
}catch(ArithmeticException e) {
	System.out.println("rest of code");
}
	}

}
